lib/src/arithmetic/multiplier.dart

```diff
@@ -240,13 +258,19 @@ abstract class MultiplyAccumulate extends Module {
   /// signed or unsigned operation, overriding the [signedAddend] static
   /// configuration.
   MultiplyAccumulate(Logic a, Logic b, Logic c,
-      {this.signedMultiplicand = false,
+      {Logic? clk,
+      Logic? reset,
+      Logic? enable,
+      this.signedMultiplicand = false,
       this.signedMultiplier = false,
       this.signedAddend = false,
       Logic? selectSignedMultiplicand,
       Logic? selectSignedMultiplier,
       Logic? selectSignedAddend,
       super.name}) {
+    this.clk = (clk != null) ? addInput('clk', clk) : null;
```

Comment:
doesn't this similar code also need to be removed from the `CompressionTreeMultiplyAccumulate` with super calls instead?